go.mod

```diff
@@ -2,4 +2,7 @@ module github.com/tablelandnetwork/wasm-sqlparser
 
 go 1.16
 
-require github.com/tablelandnetwork/sqlparser v0.0.0-20230328132500-785ebca8e351
+require (
+	github.com/hashicorp/errwrap v1.1.0 // indirect
```

Comment:
I did `go get -u github.com/tablelandnetwork/sqlparser` and it included this line.  Anyone know why?

Reply:
Not sure. Run `go mod tidy` just to make sure things are fine.